_data-prepper/pipelines/configuration/processors/key-value.md

```diff
@@ -42,4 +44,4 @@ Content will be added to this section.
 
 ## Metrics
 
-Content will be added to this section. --->
\ No newline at end of file
+Content will be added to this section. --->
```

Comment:
Is this placeholder to update text before merging this PR?

Reply:
This is from existing text. I am not planning to change this.